### DISCUSSION TITLE
Multiple languages for one document support

### DISCUSSION THREAD

**Discussion by dmpanch** [outline/outline#3218] [Ideas]
Confluence has a [handy plugin](https://help.k15t.com/scroll-translations/latest/scroll-translations-121308300.html) allows you to have multiple localizations of the same document.

When changing the document in a main language, the rest can be outdated automatically, which allows to see how up-to-date the document in a particular language is.

**Comment by mrpf-me** [comment]
Very useful! I would be very happy about an implementation!
